code/mob/living/silicon/robot.dm

```diff
@@ -260,6 +260,11 @@
 			src.remove_syndicate("death")
 		src.borg_death_alert()
 		src.eject_brain(fling = TRUE) //EJECT
+		if (length(src.clothes))
+			for (var/A in src.clothes)
+				var/obj/item/cloth_item = src.clothes[A]
+				if (cloth_item)
+					cloth_item.set_loc(src.loc)
```

Comment:
how would this be null?

Reply:
removing the null chef, it's indeed unnecessary